### ISSUE TITLE
TRO/22/03 further issues

### ISSUE THREAD

**Issue by SeanC75** [ProjectCentreLimited/TOMs#42]
There are some other issues that cropped up while drafting, that will require your assistance.
 
The following happened when I drafted a shared use space on Hillside Street, no bay was created, just a line:
 
![image](https://user-images.githubusercontent.com/115094257/199214410-906cf43e-83dc-4886-904f-a77c633d1156.png)

 
 
Plan:
 
 
![image](https://user-images.githubusercontent.com/115094257/199214428-c076cf49-fb79-46f1-b180-f2f583474553.png)

 
Laichfield, an oddly shaped which can’t be deleted, was created:
 
 
 
Plan:
 
![image](https://user-images.githubusercontent.com/115094257/199214366-0aa6a677-f7e5-42a3-af8b-93a7f619436f.png)

**Comment by Djedouas** [comment]
Hillside street issue has been fixed already: 

![image](https://user-images.githubusercontent.com/34267385/199933693-66697439-9e31-40b6-83ad-5203f950a879.png)

or maybe I misunderstood the question?

---

Laichfield issue is just a matter of placing the cursor on the right spot to be able to select the restriction:

![image](https://user-images.githubusercontent.com/34267385/199935356-6bfb52f5-3bf2-4432-9524-41ccfee361e5.png)

I used the *edit restriction* tool to see that the odd shape comes from the last vertex drawn:

![image](https://user-images.githubusercontent.com/34267385/199935650-7094ffde-1fb9-432d-8d21-4a7b7b19efc1.png)

which I deleted:

![image](https://user-images.githubusercontent.com/34267385/199935830-f146d1c5-1d3a-450d-bc9f-59c972881e02.png)

If you want to delete the restrictions, try to select them in their upper left part.

This is an issue of the *select restriction tool* which has to be addressed in a dedicated development, as this tool has not been stabilized nor optimized yet (I think that it needs a re-writing).
